backend/routers/user.py

```
@@ -18,36 +18,55 @@
+    return _sessions_schema_cache
+
+def _clear_sessions_schema_cache():
+    """Clear the sessions schema cache (useful for testing or after schema changes)"""
+    global _sessions_schema_cache
+    _sessions_schema_cache = None
+    logger.info("Sessions schema cache cleared")
+
 def get_user_id_from_token(request: Request) -> str | None:
     """Extract user ID from JWT token - OPTIONAL"""
     try:
         return JWTHandler.get_user_id_from_token(request)
```

Ruff (0.12.2)

Undefined name `JWTHandler`

(F821)